rfcs/react-components/components/combobox-spacebar.md

```diff
@@ -0,0 +1,115 @@
+- Defaulting to `true` means that often-missed use cases like spaces in translated strings or user-generated options won't cause issues
+
+#### Cons
+
+- Defaulting to `true` means the simplest/most basic use case is not the default
+- Defaulting to `true` means the Combobox space behavior is different from Dropdown by default, which primarily affects keyboard and screen reader users.
+- Providing a prop to alter the space key behavior potentially means inconsistent behavior within an app, or across related apps
+- This behavior is not aligned with the results of the user study
+
+### 2. `allowSpaceCharacter` prop, default to `false`
+
+This is the same as (1), but with a different default. The Pros/Cons list is essentially the same, but reversed.
+
+### 3. Always allow space to insert a character
+
+We would keep the behavior where if the user is actively interacting with the options in the dropdown listbox, space will select. Otherwise, space will insert a character in the input.
+
+#### Pros
+
+- Authors don't need to try to figure out what the ideal space key behavior is on a case-by-case basis / less authoring overhead for an already complex component
+- Consistent behavior across all apps using Fluent
+
+#### Cons
+
+- There is no way to opt into space-to-select behavior
+
+### 4. Always allow space to insert a character on single-select Comboboxes, always select on multi-select comboboxes
+
+Since options visually look like they have check marks and semantically use `aria-checked` instead of `aria-selected`, there is a stronger expectation of space-to-select. Additionally, multiselect comboboxes are already hitting a limit of too much complexity for one component, and most use cases would benefit from using either the multiselect Dropdown or single-select Combobox + tags instead. Opting into a simpler, more intuitive but less feature-rich UX might result in a less error-prone UX in this variant in particular.
+
+#### Pros
+
+- Authors don't need to figure out ideal space key behavior on a case-by-case basis
+- Consistent behavior across all apps using Fluent
+- The variant with the strongest expectation of space-to-select keeps that behavior
+
+#### Cons
+
+g
+
+- There is no way to opt into space-to-select on single-select combos, or space-to-type on multiselect combos
+
+### 5. Provide a context that sets spacebar default behavior
+
+In this option, we would still have an `allowSpaceCharacter` prop with one of the defaults in (1) or (2), but also provide a context that allows an entire app to change the default for all Comboboxes within it.
+
+#### Pros
+
+- Allows an app team to choose the best behavior for all Comboboxes within it
+- Splits some of the difference between choice vs. consistency
+
+#### Cons
+
+- Requires a new top-level context
+- Still has a lack of standardization between apps, which may or may not be an issue
+- Use cases may vary within an app, which still comes with the same pros/cons of options (1) and (2)
```

Comment:
I think Option 5 suffers from the same problem as Options 1-2: it relies on the dev figuring out the right behavior in a nuanced scenario.